src/Controls/src/Core/Switch/Switch.cs

```diff
@@ -29,6 +29,13 @@ public partial class Switch : View, IElementConfiguration<Switch>, ISwitch
 				((IView)bindable)?.Handler?.UpdateValue(nameof(ISwitch.TrackColor));
 			});
 
+		/// <summary>Bindable property for <see cref="OffColor"/>.</summary>
+		public static readonly BindableProperty OffColorProperty = BindableProperty.Create(nameof(OffColor), typeof(Color), typeof(Switch), null,
```

Comment:
is 'null' the right value for default here ?

Reply:
@StephaneDelcroix, Null is used as the default value for the OnColor property as well, so I have applied the same approach here